product_configurator/models/product.py

```diff
@@ -12,6 +12,10 @@
 
 class ProductTemplate(models.Model):
     _inherit = "product.template"
+    _rec_names_search = [
+        "name",
+        "default_code",
+    ]
```

Comment:
This should be out of the box, and if it weren't it is a separate concern from the Configurator IMO.

Reply:
Thanks for having a look!
If you have a look at the whole commit https://github.com/ursais/product-configurator/pull/3/commits/e9d81b64e655fd49f113f945c3f60dda29744c56 you'll see that this is simply the new way of doing what the override of `name_search` was doing before.

I understand this is not a concern of the configurator, but I simply kept this feature in the migration, should I remove it?

Reply:
I think so. I can drop the commit.

Reply:
Rather, I amend the commit.